cmake/BuildSdl2.cmake

```diff
@@ -0,0 +1,50 @@
+# find_package(SDL2 QUIET)
+if(SDL2_FOUND)
+    message(STATUS "Using SDL2 via find_package")
+endif()
+
+# 2. Try using a vendored SDL library
+if(NOT SDL2_FOUND AND EXISTS "${CMAKE_CURRENT_LIST_DIR}/SDL/CMakeLists.txt")
```

Comment:
Why is this needed?

Reply:
It's not, this was while I was waiting to figure out how we were going to handle submodules. Same with the other issue.